rust_src/src/frames.rs

```diff
@@ -661,4 +664,23 @@ pub fn visible_frame_list() -> LispObject {
     filter_frame_list(LispFrameRef::is_visible)
 }
 
+/// Select FRAME.
+/// Subsequent editing commands apply to its selected window.
+/// Optional argument NORECORD means to neither change the order of
+/// recently selected windows nor the buffer list.
+///
+/// The selection of FRAME lasts until the next time the user does
+/// something to select a different frame, or until the next time
+/// this function is called.  If you are using a window system, the
+/// previously selected frame may be restored as the selected frame
+/// when returning to the command loop, because it still may have
+/// the window system's input focus.  On a text terminal, the next
+/// redisplay will display FRAME.
+///
+/// This function returns FRAME, or nil if FRAME has been deleted.
+#[lisp_fn(min = "1")]
+pub fn select_frame(frame: LispObject, norecord: LispObject) -> LispObject {
+    unsafe { do_switch_frame(frame.into(), 1, 0, norecord.into()) }
```

Comment:
Why are `frame` and `norecord` being cast?

Reply:
My mistake, fixed now...